packages/toc-extension/schema/plugin.json

```diff
@@ -14,7 +14,7 @@
       "type": "boolean",
       "title": "collapsibleNotebooks",
       "description": "When set to true (default), when a header is collapsed in the ToC the corresponding section in the notebook is collapsed as well and vice versa.",
```

Comment:
```suggestion
      "description": "When set to true, when a header is collapsed in the ToC the corresponding section in the notebook is collapsed as well and vice versa.",
```

Reply:
Good catch! Thank you!

Reply:
I think this did not make it, we still have the (default).

Reply:
Sorry about that. That change must have gotten lost in the rebase. Whats the best way to fix that? New PR with just that change?